web/src/beta/components/fields/URLField/index.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useCallback, useState } from "react";
+
+import Button from "@reearth/beta/components/Button";
+import Property from "@reearth/beta/components/fields";
+import TextInput from "@reearth/beta/components/fields/common/TextInput";
+import useHooks from "@reearth/beta/features/Assets/AssetsQueriesHook/hooks";
+import { Asset } from "@reearth/beta/features/Assets/types";
+import { useManageAssets } from "@reearth/beta/features/Assets/useManageAssets/hooks";
+import ChooseAssetModal from "@reearth/beta/features/Modals/ChooseAssetModal";
+import { useT } from "@reearth/services/i18n";
+import { useWorkspace } from "@reearth/services/state";
+import { styled } from "@reearth/services/theme";
+
+export type Props = {
+  value?: string;
+  onChange?: (value: string | undefined) => void;
+  name?: string;
+  description?: string;
+  fileType?: "Asset" | "URL";
+  assetType?: "Image" | "File";
```

Comment:
Make all values lowercase and you can simplify your conditionals in PropertyFields 👍🏻

Reply:
done